variables.tf

```diff
@@ -19,7 +19,7 @@ variable "region" {
 }
 
 variable "project-name" {
-  default = "corded-skill-214716"
+  default = ""
```

Comment:
The variable shouldn't have a default value. Here the project name will be bound to the empty string, which is incorrect.